query/src/storages/fuse/statistics/reducers.rs

```diff
@@ -72,38 +66,39 @@ pub fn reduce_block_stats<T: Borrow<ColumnsStatistics>>(
                 in_memory_size += col_stats.in_memory_size;
             }
 
-            // TODO panic
-            let data_type = schema.field((*id) as usize).data_type();
-
-            let mut min = DataValue::Null;
-            let mut max = DataValue::Null;
+            let mut min = min_stats[0].clone();
+            let mut max = max_stats[0].clone();
 
-            let field = schema.field((*id) as usize);
             // TODO
             // for some data types, we shall balance the accuracy and the length
             // e.g. for a string col, which max value is "abcdef....", we record the max as something like "b"
-            let min_column = data_type.create_column(&min_stats)?;
-            let max_column = data_type.create_column(&max_stats)?;
-
-            let mins = eval_aggr(
-                "min",
-                vec![],
-                &[ColumnWithField::new(min_column.clone(), field.clone())],
-                min_column.len(),
-            )?;
-            let maxs = eval_aggr(
-                "max",
-                vec![],
-                &[ColumnWithField::new(max_column.clone(), field.clone())],
-                max_column.len(),
-            )?;
-
-            if mins.len() > 0 {
-                min = mins.get(0);
-            }
-            if maxs.len() > 0 {
-                max = maxs.get(0);
-            }
+            min_stats
```

Comment:
@sundy-li need your help.  I am not sure if replacing `eval_aggr` with "manually comparison" is proper here

Reply:
I think it's ok.

Reply:
got it